app/views/general/_frontpage_how_it_works.html.erb

```diff
@@ -0,0 +1,45 @@
+
+<div class="homepage-how-it-works" id="learn-more">
+  <div class="row">
+    <div class="homepage-how-it-works__hiw-content">
+      <div class="hiw-content__headings">
+        <h2><%= _("How it works") %></h2>
+        <p class="hiw__subtitle">
+        <%= _("You have the <strong>right</strong> to request information from any publicly-funded body, and get answers.
+           {{site_name}} helps you make a Freedom of Information request. It also publishes all requests online.", :site_name => site_name) %>
```

Comment:
Long strings should be terminated before 80 characters and continued on the next line:

```erb
        <p class="hiw__subtitle">
          <%= _("You have the <strong>right</strong> to request information " \
                "from any publicly-funded body, and get answers. " \
                "{{site_name}} helps you make a Freedom of Information " \
                "request. It also publishes all requests online.",
                :site_name => site_name) %>
        </p>
```

Also a typo:

```diff
- helps you make a Freedom of Information request
+ helps you make a Freedom of Information requests
```